payments/views.py

```diff
@@ -38,7 +38,7 @@ def get(self, request, *args, **kwargs):
 
 
 class WorkshopList(ListView):
-    queryset = Workshop.objects.filter(sales_open=True)
+    queryset = Workshop.objects.filter(draft=False)
```

Comment:
I'll port this over to the private workshop and rates PR since the queryset is now dynamic.